daemon/inertiad/webhook.go

```diff
@@ -34,6 +34,21 @@ func webhookHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// specialized handler for docker webhooks
+func dockerWebhookHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
any reason to make this separate from `webhookHandler`? I had imagined this would be part of that, though for now its fine for it to be separate (functionality not in place)

Reply:
I had considered that and it'd be easy to make it a part of `webhookHandler` - json unmarshal would just fail if the body didn't match up, but then I realized that I wouldn't be able to satisfy the current `Payload` interface that we are returning. Make an even more generic interface?

Reply:
Oh you're right, hm... do you have any immediate ideas for making the interface more generic? If not, I think this is fine as is until we implement #201

Reply:
Not really, but I didn't give it a lot of thought since like you mentioned, the Docker webhook functionality doesn't exist. I think this could be enough for now. In this PR, I've simply added the support for future functionality for Docker webhooks 😎

Reply:
Sounds good 😁